### ISSUE TITLE
Test Results tab shows 0% passed on sucessful execution of tests

### ISSUE THREAD

**Issue by IvoLimmen** [apache/netbeans#6181]
### Apache NetBeans version

Apache NetBeans 18

### What happened

* I select a specific test and execute the test (Ctrl+F6 or right click on the file and select 'Test File').
* Output pane shows progress
* After execution the test results panel focuses and shows "Tests passed: 0.00%" with underneath "No tests executed. (0.0 s)"

Project is very large but has the junit libraries available.

+- org.junit.jupiter:junit-jupiter-engine:jar:5.6.3:test
[INFO] |  +- org.apiguardian:apiguardian-api:jar:1.1.0:test
[INFO] |  \- org.junit.platform:junit-platform-engine:jar:1.6.3:test
[INFO] +- org.junit.jupiter:junit-jupiter-api:jar:5.6.3:test
[INFO] |  +- org.opentest4j:opentest4j:jar:1.2.0:test
[INFO] |  \- org.junit.platform:junit-platform-commons:jar:1.6.3:test
[INFO] +- com.tngtech.archunit:archunit:jar:0.23.1:test
[INFO] |  \- org.slf4j:slf4j-api:jar:1.7.36:compile
[INFO] +- org.junit.platform:junit-platform-launcher:jar:1.6.3:test
[INFO] +- org.junit.jupiter:junit-jupiter-params:jar:5.6.3:test

### How to reproduce

* Select a specific test and execute the test (Ctrl+F6 or right click on the file and select 'Test File').
* After execution the test results panel focuses and shows "Tests passed: 0.00%" with underneath "No tests executed. (0.0 s)"

### Did this work correctly in an earlier version?

No / Don't know

### Operating System

Ubuntu 23.04 (6.2.0-24-generic #24-Ubuntu SMP PREEMPT_DYNAMIC Fri Jun 16 12:03:50 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux)

### JDK

OpenJDK 64-Bit Server VM Temurin-11.0.19+7 (build 11.0.19+7, mixed mode)

### Apache NetBeans packaging

Apache NetBeans binary zip

### Are you willing to submit a pull request?

No

**Comment by Atrament666** [comment]
I had the same problem, unchecking 'Print Maven output logging level' in menu Tools - Options - Java - Maven solved it for me.

**Comment by IvoLimmen** [comment]
Unfortunately this does not solve the issue for me.

**Comment by sroeper** [comment]
Same for me, after updating to Netbeans 18 the Test Result Window is useless (at least in maven projects) but pops up every time after running a test.

**Comment by neilcsmith-net** [comment]
Please share a basic project that causes this issue for you, thanks!

**Comment by sroeper** [comment]
[testresultwindow.zip](https://github.com/apache/netbeans/files/12312274/testresultwindow.zip)
Steps to reproduce:
1 Clean and build
2. Goto to TestresultwindowTest.java
3. Run the test file

Expected Result:
Test Results window opens and show one green test

Actual Result:
Test Results window opens and show Tests passed 0%, No tests executed

**Comment by sroeper** [comment]
@neilcsmith-net Is still the same under NB19, there is also an older issue (https://github.com/apache/netbeans/issues/4734) describing the same behaviour.

**Comment by neilcsmith-net** [comment]
@sroeper just tried your project between you linking and commenting.  Works as expected for me (on Ubuntu with JDK 20 and bundled Maven 3.9.3).

**Comment by sroeper** [comment]
@neilcsmith-net That sounds funny, I just run the test with NB19 and the described behaviour is unchanged. How do you execute the Test? I just open the test file and hit Command-F6, but get the same result by right click - Test file.

**Comment by sroeper** [comment]
@neilcsmith-net I read that you have a M1 local, can you please test on that machine? May be it is another mac only bug.

**Comment by neilcsmith-net** [comment]
@sroeper I have, although only with Rosetta.

However, have you changed any of the Maven settings from their defaults.  In particular, from https://github.com/apache/netbeans/issues/6181#issuecomment-1636480236 - do you have `Print Maven output logging level` enabled?

**Comment by sroeper** [comment]
I checked again, and unchecking 'Print Maven output logging level' in menu Tools - Options - Java - Maven in NB19 now shows the test results window again **with** content. I still used the bundled maven version (3.9.3).

But when I use my own maven installation (maven 3.8.6) it is still empty. That was what I already tried on NB18 without success. To double check that I installed maven 3.9.4 locally and used that in Netbeans. This combination also works.

So in summary I have to disable 'Print Maven output logging level' **and** use the a maven version >= 3.9.3.

But that is, umm, at least confusing.

**Comment by IvoLimmen** [comment]
I upgraded to Netbeans 19. Issue still remains. Like @sroeper I also tried the bundled (3.9.3) version and my local version (3.9.5) both with the 'Print Maven output logging level' off and on. But no combination made it work on my Ubuntu 23.04.

**Comment by Saljack** [comment]
I found that it works in a simple project but it does not work with other complex project. The surefire output is the same and both are Spring projects. Can you try to make simple maven project with one test and run it? Just to confirm it.

**Comment by IvoLimmen** [comment]
@Saljack just tested that theory on a small project:

SLOC	Directory	SLOC-by-Language (Sorted)
4391    XXX-client-api java=3698,xml=693
307     XXX-client-impl java=184,xml=123
300     XXX-stub java=238,xml=62
289     top_dir         xml=289
185     XXX-client-mock xml=102,java=83
0       src_license     (none)
0       target          (none)


Totals grouped by language (dominant language first):
java:          4203 (76.81%)
xml:           1269 (23.19%)

And it still does not show me the test result panel.

**Comment by Saljack** [comment]
@IvoLimmen I would bet that all have the same parent pom (and then there is a problem with this parent). Try to create a new project and add there only JUnit dependency and one unit test.

**Comment by IvoLimmen** [comment]
I have tested it now with other projects (those in my own GitHub profile) and they seem to work; so I guess you might be right about the parent pom corrupting things.

But then I would still like to know what it is that causes this issue.
